Project-I-Social-Card/social-card/src/components/FooterComponents/Footer.js

```diff
@@ -1,19 +1,64 @@
 import React from 'react';
 import './Footer.css';
 
-const Footer = () => {
-    return ( 
-        <div className = "Footer" >
-       <img src = "https://i.imgur.com/bDi6wmh.png"
-       alt = "" ></img> 
-       <img src = "https://i.imgur.com/qNI4CTC.png"
-       alt = "" ></img> 
-       <img src = "https://i.imgur.com/CWkoyIk.png"
-       alt = "" ></img> 
-       <img src = "https://i.imgur.com/g45H4kZ.png"
-       alt = "" ></img>
-        </div>
-    );
-};
+class Footer extends React.Component {
```

Comment:
Found some example code using props and state. I decided to go back to original because i didn't quite get it. I want to read up more on that tomorrow. went back to the lines 4-17